src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/DynamicMethod.CoreCLR.cs

```diff
@@ -0,0 +1,306 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Runtime.Loader;
+using System.Text;
+using System.Threading;
+using static System.Runtime.CompilerServices.RuntimeHelpers;
+
+namespace System.Reflection.Emit
+{
+    public sealed partial class DynamicMethod : MethodInfo
+    {
+        private RuntimeType[] m_parameterTypes;
+        internal IRuntimeMethodInfo? m_methodHandle;
+        private RuntimeType m_returnType;
+        private DynamicILGenerator? m_ilGenerator;
+        private DynamicILInfo? m_DynamicILInfo;
+        private bool m_fInitLocals;
+        private Module m_module;
```

Comment:
Both CoreCLR and Mono have `RuntimeModule`. Why does this need to be downgraded to `Module`?